xcell/cls/cl.py

```diff
@@ -48,6 +48,12 @@ def get_ell_cl(self):
             self.get_cl_file()
         return self.ell, self.cl
 
+    def get_n_cls(self):
```

Comment:
Nice.

Can you replace calls to `get_dof_tracers`  in `to_sacc.py` with this method? https://github.com/xC-ell/xCell/blob/d37b89b7fd5c147d02bde9b4f70dcea937c4a774/xcell/cls/to_sacc.py#L192

Reply:
Well, the input of both functions is not the same, so instead I've streamlined `get_dof_tracers`